docs/src/templates/pages/styleguide/ui-templates/index.hbs

```diff
@@ -8,8 +8,9 @@ navsub: ui-templates
 navactive: index
 ---
 
-- [Content Page – Cookie Policy]({{ baseUrl }}/styleguide/ui-templates/content-pages/cookie-policy)
-- [Content Page – Privacy Policy ]({{ baseUrl }}/styleguide/ui-templates/content-pages/privacy-policy)
+- [Menu]({{ baseUrl }}/styleguide/ui-templates/menu/)
+- [Content Pages]({{ baseUrl }}/styleguide/ui-templates/content-pages/)
```

Comment:
Prob best to re-include content pages under their own title.  Makes sense to just keep all of that on one page.

Up to you if you'd like to keep menu pages as a separate page index…probably depends how many variations there will be.